server/http/server.go

```diff
@@ -89,6 +83,19 @@ func (s *Server) HandleSubscriptions() error {
 	return nil
 }
 
+func optionsHandler(h http.Handler) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		if r.Method == "OPTIONS" {
```

Comment:
same here. i would use predefiend constant in net/http pkg. `http.MethodOptions`

Reply:
net/http package does not define constants for content type (https://golang.org/pkg/net/http). Is this perhaps from diff http package?

Reply:
they have method and status.

https://golang.org/src/net/http/method.go
https://golang.org/src/net/http/status.go

Reply:
added the use of constants